src/index.html

```diff
@@ -9,6 +9,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#" class="logo_link">
+        <img 
+          src="images/logo.png" 
+          alt="logo" 
+          class="logo flex_item"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
here we can add an empty line between multiline siblings
```suggestion
      </a>
      
      <nav class="nav">
```